robot-core/src/main/java/org/obolibrary/robot/Template.java

```diff
@@ -559,33 +559,23 @@ private void addTable(List<List<String>> rows) throws Exception {
 
   /** Add the labels from the rows of the template to the QuotedEntityChecker. */
   private void addLabels() {
-    // If there's no label column, we can't add labels
-    if (labelColumn == -1) {
-      return;
-    }
     for (List<String> row : tableRows) {
       String id = null;
-      if (idColumn != -1) {
-        try {
-          id = row.get(idColumn);
-        } catch (IndexOutOfBoundsException e) {
-          // ignore
-        }
-      }
-
-      String label = null;
       try {
-        label = row.get(labelColumn);
+        id = row.get(idColumn);
       } catch (IndexOutOfBoundsException e) {
         // ignore
       }
 
-      if (idColumn != -1 && id == null) {
+      if (id == null) {
         continue;
       }
 
-      if (id == null || label == null) {
-        continue;
```

Comment:
Removing this does change the behaviour of the loop considerably; but it makes sense, since you want to add "id in case entity label is missing".

Reply:
Yes, that's the decision.